source/Halibut/Transport/Protocol/WebSocketStream.cs

```diff
@@ -166,14 +166,15 @@ public override long Position
             set => throw new NotImplementedException();
         }
 
-        void SendCloseMessage()
+        async Task SendCloseMessage()
```

Comment:
Could you copy past this method to keep the sync behaving just like it used to.

Reply:
I figured that it was logically the same. But I can put it back to be safe 🙂